src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3617,6 +3617,30 @@
     </Field>
     <Field Name="EqualsValue" Type="EqualsValueClauseSyntax" Optional="true"/>
   </Node>
+  <Node Name="ExtensionDeclarationSyntax" Base="TypeDeclarationSyntax">
+    <TypeComment>
+      <summary>Extension container syntax.</summary>
+    </TypeComment>
+    <Kind Name="ExtensionDeclaration" />
+    <Field Name="AttributeLists" Type="SyntaxList&lt;AttributeListSyntax&gt;" Override="true"/>
+    <Field Name="Modifiers" Type="SyntaxList&lt;SyntaxToken&gt;" Override="true"/>
+    <Field Name="Keyword" Type="SyntaxToken" Override="true">
+      <ContextualKind Name="ExtensionKeyword"/>
+    </Field>
+    <Field Name="TypeParameterList" Type="TypeParameterListSyntax" Optional="true" Override="true"/>
+    <Field Name="ParameterList" Type="ParameterListSyntax" Optional="true" Override="true" />
+    <Field Name="ConstraintClauses" Type="SyntaxList&lt;TypeParameterConstraintClauseSyntax&gt;" Override="true"/>
+    <Field Name="OpenBraceToken" Type="SyntaxToken" Override="true" Optional="true">
+      <Kind Name="OpenBraceToken"/>
+    </Field>
+    <Field Name="Members" Type="SyntaxList&lt;MemberDeclarationSyntax&gt;" Override="true"/>
+    <Field Name="CloseBraceToken" Type="SyntaxToken" Override="true" Optional="true">
+      <Kind Name="CloseBraceToken"/>
+    </Field>
+    <Field Name="SemicolonToken" Type="SyntaxToken" Optional="true" Override="true">
```

Comment:
Why is the semicolon token allowed here? It's not in the speclet.

Reply:
I kept it for consistency and simplicity with the other declaration kinds. Will [update](https://github.com/dotnet/csharplang/pull/9090) the speclet.